modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -3218,6 +3218,65 @@ definition {
 			actionLabel = "Update Action Label");
 	}
 
+	@description = "On Commerce Product, add / view update action on System Objects using System Objects"
+	@priority = 5
+	test CanUpdateEntryWithActionOnCommerceProduct {
+		property portal.acceptance = "true";
+
+		task ("Given a minium site and the Commerce Product with an Action (Add) when update entry") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+			ApplicationsMenu.gotoPortlet(
+				category = "Object",
+				panel = "Control Panel",
+				portlet = "Objects");
+
+			ObjectPortlet.selectSystemObject(label = "Commerce Product");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionLabel = "Custom Action",
+				active = "true",
+				fieldName = "Name",
+				inputMethod = "true",
+				inputValue = "Shoes",
+				thenAction = "Update an Object Entry",
+				whenAction = "On After Add");
+		}
+
+		task ("When add a new Product") {
+			CommerceNavigator.gotoPortlet(
+				category = "Product Management",
+				portlet = "Products");
```

Comment:
Test it here: `CommerceProducts.openProductsAdmin();`

I'm not sure if it works, but could save time on CI with access via URL.